homeassistant/components/sonos/speaker.py

```diff
@@ -177,6 +177,7 @@ def __init__(
         # Device information
         self.mac_address = speaker_info["mac_address"]
         self.model_name = speaker_info["model_name"]
+        self.uid = speaker_info["uid"]
```

Comment:
We could probably use `self.soco.uid`, but I think its better to do this since SoCo doesn't follow the pattern of not doing I/O when accessing properties.